src/shared/bg_public.h

```diff
@@ -320,11 +320,11 @@ void Pmove( pmove_t *pmove );
 enum statIndex_t
 {
   STAT_HEALTH,
-  STAT_ITEMS,
+  STAT_ITEMS,      // bitfield of all possible upgrades, probably only for humans.
   STAT_ACTIVEITEMS,
-  STAT_WEAPON,     // current primary weapon
+  STAT_WEAPON,     // current primary weapon. Works for humans and aliens. For aliens, is related to STAT_CLASS.
   STAT_MAX_HEALTH, // health limit
-  STAT_CLASS,      // player class
+  STAT_CLASS,      // player class. Human's armor or alien's form. Can also get used to guess team.
```

Comment:
The usual way to get the team from playerstate is with PERS_TEAM, so I'd remove the bit about guessing the team.

Reply:
I think I've seen it in code, using STAT_CLASS to guess species... I've updated the comment to say "don't do that, use PERS_TEAM instead".